apps/condo/domains/common/components/BaseSearchInput/index.tsx

```diff
@@ -1,16 +1,16 @@
-import { isEmpty } from 'lodash'
-import isFunction from 'lodash/isFunction'
-import React, { Dispatch, SetStateAction, useCallback, useEffect, useMemo, useState } from 'react'
 import Select, { CustomSelectProps } from '@condo/domains/common/components/antd/Select'
+import { Loader } from '@condo/domains/common/components/Loader'
+import { TrackingEventPropertiesType } from '@condo/domains/common/components/TrackingContext'
+import { isNeedToLoadNewElements } from '@condo/domains/common/utils/select.utils'
+import { useIntl } from '@open-condo/next/intl'
+import { isEmpty } from 'lodash'
```

Comment:
we use to sort imports

Reply:
Already [sorted](https://github.com/open-condo-software/condo/pull/2774/commits/cc94361a9d40e87134339aa826fc45f1576b1cc2) alphabetically.